### PR TITLE
Added instruction how to use different algorithms for signing SAML me…

### PR DESCRIPTION
**Pull request by MikhailStrokov**
Added instruction how to use different algorithms for signing SAML messages

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
 
 Also, you can observe the details of the implementation in the corresponding [demo project](https://git.haulmont.com/app-components/saml-addon-demo).
 
+## 4.2. Setup signing method for SAML messages <a name="setup-signing-method"></a>
+
+By default opensaml spring component uses sha1 digest algorithm for signing SAML messages. The most convenient way to use different signing message
```
Comment (zhenyazb) [inline Readme.md]: By default **,** **OpenSAML Spring** component uses **SHA1** digest algorithm for signing SAML messages. The most convenient way to use different signing message**s**

File: Readme.md [hunk 2]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
...
+is create class in `WEB` module with addition changes in SecurityContext.
```
Comment (zhenyazb) [inline Readme.md]: is **to** create **a** class in **the** `web` module with **additional** changes in `SecurityContext`.

File: Readme.md [hunk 3]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
...
+        configuration.setSignatureCanonicalizationAlgorithm(SignatureConstants.ALGO_ID_C14N_EXCL_OMIT_COMMENTS);
+        configuration.setSignatureHMACOutputLength(null);
+        configuration.setSignatureReferenceDigestMethod(SignatureConstants.ALGO_ID_DIGEST_SHA256);
+    }
+}
+```
+</details>
+
+
+Declare bean in `saml-dispatcher-spring.xml`
```
Comment (zhenyazb) [inline Readme.md]: Declare bean in **the** `saml-dispatcher-spring.xml`.

File: Readme.md [hunk 4]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
...
+
+```xml
+<!-- Initialization of OpenSAML library-->
+<bean id="samlBootstrap" class="org.springframework.security.saml.SAMLBootstrap"/>
+...
+<bean id="securityConfiguration" class="com.haulmont.addon.saml.web.SecurityConfiguration" init-method="initialize"
+          depends-on="samlBootstrap"/>
+```
+Basic configuration initilized in class `org.springframework.security.saml.SAMLBootstrap`.
```
Comment (zhenyazb) [inline Readme.md]: Basic configuration **is initialized** in **the** `org.springframework.security.saml.SAMLBootstrap` **class**.

File: Readme.md [hunk 5]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
...
+
+```xml
+<!-- Initialization of OpenSAML library-->
+<bean id="samlBootstrap" class="org.springframework.security.saml.SAMLBootstrap"/>
+...
+<bean id="securityConfiguration" class="com.haulmont.addon.saml.web.SecurityConfiguration" init-method="initialize"
+          depends-on="samlBootstrap"/>
+```
+Basic configuration initilized in class `org.springframework.security.saml.SAMLBootstrap`. 
+To be sured that security config initialized and not overrides your changes set depends-on attribute with value of bean id of class `org.springframework.security.saml.SAMLBootstrap`.
```
Comment (zhenyazb) [inline Readme.md]: To **make sure** that security config **is** initialized and **does not override** your changes set depends-on attribute with **the** value of **the** bean id of **the** `org.springframework.security.saml.SAMLBootstrap` **class**.

File: Readme.md [hunk 6]
```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
...
+ 
+All supported signing methods declared in class `org.opensaml.xml.signature.SignatureConstants`
```
Comment (zhenyazb) [inline Readme.md]: All supported signing methods **are** declared in **the** `org.opensaml.xml.signature.SignatureConstants` **class**.

File: Readme.md (hunk 1 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

File: Readme.md (hunk 3 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

File: Readme.md (hunk 2 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

File: Readme.md (hunk 4 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

File: Readme.md (hunk 5 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

File: Readme.md (hunk 6 above)
Comment (MikhailStrokov) [reply Readme.md]: fixed

**Pull request merged**